requirements-full.txt

```diff
@@ -14,9 +14,9 @@ psutil==5.8.0
 lxml==4.6.5
 opencv-python==4.5.1.48
 spikeextractors==0.9.9
-spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git
+spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git@12b7863684b705e3d0ae8165bb3009143c70530c
 neo==0.10.2
-roiextractors @ git+https://github.com/catalystneuro/roiextractors.git
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@10265be4a1eaece7461296a2877cc11c84bf87dc
```

Comment:
it would be better to cut a release on pypi and pin to that

Reply:
For ROIExtractors we can control that of course, but not for SpikeInterface

Reply:
We're mostly debugging the terrible MacOS+Py3.8 combination of git installation right now

Reply:
what's terrible about it?

Reply:
Producing unexpected behavior never seen before unique to that combination of os and py version, we think we may have pinned it down

Reply:
Can confirm fixed, all Mac CI are past that installation point now